libs/model/src/models/xp_log.ts

```diff
@@ -0,0 +1,23 @@
+import { XpLog } from '@hicommonwealth/schemas';
+import Sequelize from 'sequelize';
+import { z } from 'zod';
+import type { ModelInstance } from './types';
+
+export type XpLogInstance = ModelInstance<z.infer<typeof XpLog>>;
+export type XpLogModelStatic = Sequelize.ModelStatic<XpLogInstance>;
+
+export default (sequelize: Sequelize.Sequelize) =>
+  <XpLogModelStatic>sequelize.define<XpLogInstance>(
+    'XpLog',
+    {
+      user_id: { type: Sequelize.INTEGER, primaryKey: true },
+      created_at: { type: Sequelize.DATE, primaryKey: true },
```

Comment:
Is this value going to be managed by us?

Reply:
yes, we don't need updated_at here, this is append only

Reply:
Right but what I mean is, do we need to be providing the date manually for any reason or can we just have Sequelize manage it (and disable `updated_at`).

Reply:
didn't know you can only enable created_at with sequelize... can you show me one example and I'll adjust it

Reply:
![image](https://github.com/user-attachments/assets/15887767-a522-40f7-845e-4be94d7541cc)
```
createdAt: 'created_at`,
updatedAt: false,
```